src/client/index.tsx

```diff
@@ -2,7 +2,16 @@ import * as React from 'react';
 import { render } from 'react-dom';
 import App from './routes/Routes';
 import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/store/configureStore';
```

Comment:
store should be exported as singleton object

Reply:
I think the name is misleading here. It should be configured store and I am calling the same which passing the value to store.